### ISSUE TITLE
Reconsider "Does your MP represent you?" wording

### ISSUE THREAD

**Issue by abimysoc** [mysociety/theyworkforyou#875]
It's the wrong question to ask. Two reasons:

1. The answer to "Does your MP represent you?" is "Yes, they do". Representing constituents is basically the definition of an MP.

2. It's too easily misparsed as "_Who_ represents you?" Yes, you can use TWFY to look up who your MP is, but that's underselling all the other features.

I'd suggest being bolder with the wording, more along the lines of "Is your MP doing their job properly?"

**Comment by MyfanwyNixon** [comment]
I see your point, but 'represent' can also mean 'represent your views and opinions' which is how I meant it here. Yes your MP is **supposed** to represent you, but do they **actually**?

Having said that, I wouldn't object to wording that makes that concept clearer. Later in the copy we use the phrase 'what exactly do they do in your name?': we could also use "What does your MP do in your name?", or similar, in the main text.

Might be an interesting one to user test or A/B split etc.

**Comment by abimysoc** [comment]
Yeah, the double meaning of 'represent' isn't helping. A/B/N testing a selection of phrases would be fun if we can. I offer "Is your MP working for you?"

**Comment by MyfanwyNixon** [comment]
A user writes:

> This is possibly a very pedantic point but it feels important to me, and possibly to other people.

>I know who the MP is for the constituency in which I live, but arriving on the site and being greeted with a banner asking if she represents me is just painful.  I didn't vote for her, I wouldn't vote for her, when I write to her about any issues if I get a reply at all it is a patronising potted statement of her position which is invariably the opposite of mine, she has never shown any respect for my views or any indication that she would even attempt to represent me.  I am acutely aware that I am not represented in Parliament, under the present FPTP system - like 63% of the other people who voted in the last general election.

>Might it be more accurate to ask if an MP is the member for the constituency in which the site visitor lives, rather than whether they represent them?  It's erroneous and misleading to talk about most of these people as being "my MP" or "your MP", it's just a fiction to lend an illusion of democracy, most of them don't represent more than a handful of their constituents.

**Comment by akuckartz** [comment]
:+1: to the previous comment:
> it's just a fiction to lend an illusion of democracy, most of them don't represent more than a handful of their constituents.

**Comment by MyfanwyNixon** [comment]
Just to note my response, which read (in part):

> The wording was chosen to fit in with the site's name of 'TheyWorkForYou' and, like the name, I suppose it is intended to be slightly provocative too. If the wording is provoking the response 'No, she doesn't!', then you might say that it's doing its job.
